lib/ansible/plugins/inventory/__init__.py

```diff
@@ -174,9 +182,11 @@ def parse(self, inventory, loader, path, cache=True):
                  you can ignore if this plugin does not implement caching.
         '''
 
-        self.loader = loader
         self.inventory = inventory
-        self.templar = Templar(loader=loader)
+
+        self.loader = loader
+        variables = combine_vars(load_extra_vars(loader=self.loader), load_options_vars())
```

Comment:
`load_options_vars() takes exactly 1 argument (0 given)`, but otherwise looks good to me